src/components/FetchApi.js

```diff
@@ -1,52 +1,131 @@
 import React, { Component } from 'react'
 import '../components/style.css';
+import Render from './Render';
```

Comment:
Name your components better, render is a react related keyword, name the component on the basis of what functionality it performs like UsersList, or Cards, UserCard
when you look at the file or the file name it should be descriptive enough